### PR TITLE
Changes as discussed in #4300

### PR DESCRIPTION
**Pull request by finanalyst**
The changes to this file can be seen at [new-raku pod](new-raku.finanalyst.org/language/pod)

### CODE REVIEW

File: doc/Language/pod.rakudoc [hunk 1]
```diff
@@ -295,54 +543,134 @@ C<=end para> block, any blank lines are preserved.
...
+=end code
 
 =head2 Lists
 
-=head3 Unordered lists
+Lists in Rakudoc are specified as a series of contiguous C<=item> blocks.
+No special "container" directives or other delimiters are required to enclose the entire list.
 
-Lists in Pod6 are specified as a series of C<=item> blocks.
+Note that C<=item> is just an abbreviation for C<=item1>.
```
Comment (coke) [inline doc/Language/pod.rakudoc:647]: s/just//

File: doc/Language/pod.rakudoc [hunk 2]
```diff
@@ -356,12 +684,127 @@ The three suspects are:
 =item  Sleepy
 =item  Grumpy
 
+The bulletting strategy used for different levels within a nested list is entirely up to the renderer.
```
Comment (coke) [inline doc/Language/pod.rakudoc:687]: I would rephase this to avoid bulleting. Maybe "rendering of bullets."

File: doc/Language/pod.rakudoc [hunk 3]
```diff
@@ -508,31 +1005,31 @@ B<P>, B<R>, B<T>, B<U>, B<V>, B<X>, and B<Z>.
...
 To format a text in italic enclose it in C<I< >>
-=for code :lang<rakudoc>
+=for code
 Raku is I<awesome>
 
 Raku is I<awesome>
 
 =head2 Underlined
 
 To underline a text enclose it in C<U< >>
```
Comment (coke) [inline doc/Language/pod.rakudoc:1023]: remove "a"

File: doc/Language/pod.rakudoc [hunk 4]
```diff
@@ -671,76 +1170,55 @@ B<replaceable item>, a placeholder, or a metasyntactic variable. It is
...
 
-Raku makes considerable use of the E<171> and E<187> characters.
-
 Raku makes considerable use of the E<0b10101011> and E<0b10111011> characters.
 
 Raku makes considerable use of the E<0o253> and E<0o273> characters.
 
 Raku makes considerable use of the E<0d171> and E<0d187> characters.
 
 Raku makes considerable use of the E<0xAB> and E<0xBB> characters.
```
Comment (coke) [inline doc/Language/pod.rakudoc:1208]: looks like you were working against an older version and deleted these recent additions.

**Review by coke — CHANGES_REQUESTED** [review]
I ran out of steam reviewing, so am submitting my review in progress.

This document could stand to be broken up into separate documents, perhaps.

File: doc/Language/pod.rakudoc [hunk 5]
```diff
@@ -1,28 +1,28 @@
...
-A very simple Pod6 document
+A very simple Rakudoc document
 
 =end pod
 =end code
 
 =head1 Block structure
 
-A Pod6 document may consist of multiple Pod6 blocks. There are four ways to
+A Rakudoc document may consist of multiple Rakudoc blocks (also called pod locks). There are four ways to
```
[the remark this answers is not in the document]
Comment (raiph) [reply doc/Language/pod.rakudoc:25]: "Pod locks"?

😢

File: doc/Language/pod.rakudoc [hunk 6]
```diff
@@ -1,28 +1,28 @@
 =begin pod :kind("Language") :subkind("Language") :category("reference")
 
-=TITLE Pod6
+=TITLE Rakudoc - formerly POD6
```
[the remark this answers is not in the document]
Comment (finanalyst) [reply doc/Language/pod.rakudoc:3]: agree to de-emphasise POD6

File: doc/Language/pod.rakudoc [hunk 7]
```diff
@@ -37,13 +37,13 @@ SYNOPSIS>) are reserved. Indentation of the =begin/=end lines is
 required to be the same to create a valid block; otherwise, an error
 or unexpected results will occur.
 
-=begin code :lang<rakudoc>
+=begin code
```
[the remark this answers is not in the document]
Comment (finanalyst) [reply doc/Language/pod.rakudoc:40]: The removal of `:lang` is a hack that should not have got here. I am having lots of problems getting `:allow` to work with `=code` that is then highlighted. I thought I solved it, but then it crops up again.

File: doc/Language/pod.rakudoc [hunk 8]
```diff
@@ -75,43 +75,174 @@ outside of strings. Hash keys need not be quote-delimited unless they contain
...
+=begin para B<:formatted<B I>>
+Warning: Do not immerse in water. Do not expose to bright light. Do not feed after midnight.
+=end para
+=end code
+The internal representations of these two versions are exactly the same, except that the second one
+retains the C<:formatted> option information as part of the resulting block object.
+
+Like all formatting codes, codes applied via a C<:formatted> are inherently cumulative.
+For example, if the block itself is already inside a formatting code, that formatting code will still apply,
+in addition to the extra "basis" and "important" formatting specified by C<:formatted<B I>>.
```
[the remark this answers is not in the document]
Comment (finanalyst) [reply doc/Language/pod.rakudoc:172]: I have copied exactly the wording from S26. I dont want to include `:formatted` at all.

File: doc/Language/pod.rakudoc [hunk 9]
```diff
@@ -75,43 +75,174 @@ outside of strings. Hash keys need not be quote-delimited unless they contain
...
+=begin para :margin<|>
+    |This text is flush with the (virtual) left margin of
+    |the Pod block because that margin is explicitly marked
+    |by the C<|>, as specified by the block's C<:margin<|>> option.
+=end para
+=end code
+
+The virtual margin can even be to the left of the opening delimiter, which can be convenient to guide subsequent indentations.
+For example:
+=begin code
```
[the remark this answers is not in the document]
Comment (finanalyst) [reply doc/Language/pod.rakudoc:216]: Do we even want a virtual margin. I did not understand S26 here, to be honest.

File: doc/Language/pod.rakudoc (hunk 1 above)
Comment (2colours) [reply doc/Language/pod.rakudoc:647]: Sure? Emphasizing that it isn't a first-class thing might even be good.

**Comment by finanalyst** [comment]
Closing issue as it is resolved by new RakuDoc v2 specification. All comments in this thread were considered when writing new spec.

**Pull request closed without merging**